### PR TITLE
[Metrics] Adding metrics interface for block computation effort vector

### PR DESCRIPTION
**Pull request by Tonix517**
As the first step for #3312 - now we add metrics interface for the new unweighted computation effort of each block.

### CODE REVIEW

**Review by janezpodhostnik — CHANGES_REQUESTED** [review]

File: module/metrics.go [hunk 1]
```diff
@@ -448,6 +448,9 @@ type ExecutionMetrics interface {
 	// ExecutionBlockExecuted reports the total time and computation spent on executing a block
 	ExecutionBlockExecuted(dur time.Duration, compUsed uint64, txCounts int, colCounts int)
 
+	// ExecutionBlockComputationKindExecuted reports the unweighted effort of given ComputationKind at block level
+	ExecutionBlockComputationKindExecuted(string, uint)
```
[the remark this answers is not in the document]
Comment (Tonix517) [reply module/metrics.go]: addressed

File: module/metrics/execution.go [hunk 2]
```diff
@@ -39,6 +39,7 @@ type ExecutionCollector struct {
 	readDuration                        prometheus.Histogram
 	readDurationPerValue                prometheus.Histogram
 	blockComputationUsed                prometheus.Histogram
+	blockComputationKinds               *prometheus.GaugeVec
```
[the remark this answers is not in the document]
Comment (Tonix517) [reply module/metrics/execution.go]: addressed

File: module/metrics/execution.go [hunk 3]
```diff
@@ -212,6 +213,13 @@ func NewExecutionCollector(tracer module.Tracer) *ExecutionCollector {
 		Buckets:   []float64{1000, 10000, 100000, 500000, 1000000, 5000000, 10000000},
 	})
 
+	blockComputationKinds := promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: namespaceExecution,
+		Subsystem: subsystemRuntime,
+		Name:      "block_unweighted_effort_computation_kinds",
+		Help:      "unweigthed effort of last execution block for given computation kind",
```
[the remark this answers is not in the document]
Comment (Tonix517) [reply module/metrics/execution.go]: addressed

**Review by janezpodhostnik — APPROVED** [review]

**Comment by Tonix517** [comment]
bors merge

**Comment by Tonix517** [comment]
bors retry

**Pull request merged**